compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -2533,6 +2533,99 @@ tests tools = testGroup "Data Dependencies" $
             , "--project-root", tmpDir </> "main"
             , "--target", LF.renderVersion LF.versionDev ]
 
+    , testCaseSteps "Package ids are stable across rebuilds" $ \step -> withTempDir $ \tmpDir -> do
```

Comment:
just double checking since you asked for review, this isn’t actually fixing the issue given that I only see a test addition?

Reply:
just a failing test for now